include/stumpless/target/stream.h

```diff
@@ -154,6 +155,32 @@ STUMPLESS_PUBLIC_FUNCTION
 struct stumpless_target *
 stumpless_open_stream_target( const char *name, FILE *stream );
 
+/**
+ * Sets the ANSI code to be printed in a specific target when a log is made at some severity
+ *
+ * It should be used with stdout or stderr as targets, since it is only for aesthetic purposes.
+ *
+ * **Thread Safety: MT-Safe race:name**
+ * This function is thread safe, of course assuming that escape_code is not modified by
+ * any other threads during execution.
+ *
+ * **Async Signal Safety: AS-Unsafe heap**
```

Comment:
this is both `AS-Unsafe` and `AC-Unsafe` due to lock usage, but I don't see any heap usage. Be sure you update the verbiage below the designations as well.

Reply:
Done.